CODEOWNERS

```diff
@@ -17,6 +17,9 @@ run_python_examples.sh @msaroufim
 ./reinforcement_learning/ @msaroufim 
 ./word_language_model/ @msaroufim
 
+# Forward Forward Examples
+./mnist_forward_forward/ @viveks-codes
```

Comment:
This is specific to people with write access to the repo so might be best to revert it. Happy to change this assumption if you end up being a regular contributor